lib/dialects/sqlite/query-generator.js

```diff
@@ -332,7 +332,7 @@ const QueryGenerator = {
 
   quoteIdentifier(identifier) {
     if (identifier === '*') return identifier;
-    return Utils.addTicks(identifier, '`');
+    return Utils.addTicks(Utils.removeTicks(identifier, '`'), '`');
```

Comment:
@felixfbecker Maybe instead, this should escape the identifiers instead of removing them.  I think this is what `escapeId` on line 7 of `/lib/sql-string.js` was originally meant to do.